drivers/media/i2c/Kconfig

```diff
@@ -149,6 +149,20 @@ config VIDEO_HI847
           To compile this driver as a module, choose M here: the
           module will be called hi847.
 
+config VIDEO_MIRA220
```

Comment:
These are normally in alphabetical order. (I see the Arducam drivers have got out of position during forward porting).

Annoyingly this will also move the position in the defconfigs, so perhaps I don't care enough.